app/testOnly/controllers/SessionStorageServiceController.scala

```diff
@@ -48,52 +49,79 @@ class SessionStorageServiceController @Inject()(implicit val ec: ExecutionContex
       handleShow(isAgent = true)
   }
 
+  def showByMtditid(mtditid: String): Action[AnyContent] = auth.authenticatedAction(isAgent = false) {
```

Comment:
we are not supporting requests from authenticated individuals atm, only agents permitted, but this can changes in the future